### ISSUE TITLE
Potential errors in the code.

### ISSUE THREAD

**Issue by safocl** [chevichelov/CONTROL_AMOLED#1]
https://github.com/chevichelov/CONTROL_AMOLED/blob/4bcebbfc47ce70fc887c89849c1af22fa85e7803/CONTROL_AMOLED/CONTROL_AMOLED.ino#L392 ставит 45 gpio в input, но после читается из 10 gpio (https://github.com/chevichelov/CONTROL_AMOLED/blob/4bcebbfc47ce70fc887c89849c1af22fa85e7803/CONTROL_AMOLED/CONTROL_AMOLED.ino#L395) -- это ошибка?
Ну и после подобное тоже имеется.

**Comment by chevichelov** [comment]
Нет, это что бы не случилось короткое замыкание, посмотрите на схему.
